examples/tests/qdec_callback/main.c

```diff
@@ -0,0 +1,46 @@
+#include <led.h>
+#include <qdec.h>
+#include <timer.h>
+#include <tock.h>
+
+static void qdec_callback(int qdec_num,
+                          __attribute__ ((unused)) int val,
+                          __attribute__ ((unused)) int arg2,
+                          __attribute__ ((unused)) void *ud) {
+  printf("QDEC Absolute Position: %d\n", qdec_num);
+
+  // Set LEDs based on acc value
+  // Note: LED rules arbitrarily chosen by developer
+  if (qdec_num > 0) {
+    led_on(0);
+    led_off(1);
+    led_off(2);
+  }  else if (qdec_num == 0) {
+    led_on(1);
+    led_off(0);
+    led_off(2);
+  } else if (qdec_num < 0) {
+    led_on(2);
+    led_off(0);
+    led_off(1);
```

Comment:
Generally, Tock only requires/expects platforms to have 1 LED (though many do have more). It would be nice if this could be agnostic to the number of LEDs as a test (i.e. by varying blink rate).

Reply:
This was a point of concern when I first wrote this, so this makes sense. I will update it for the next round of review.

Reply:
Ended up removing the leds from the code altogether since it did not seem relevant to the overall goal of the tests.